proxy.conf.template

```diff
@@ -15,19 +16,34 @@ server {
     # be idendified by either domain
     server_name ${PROXY_DOMAIN} ${PROXY_ORIGIN};
     root /var/www/html;
-
     listen ${PORT};
 
+    # we need a resolver configured so we can look up domains when redirecting
+    resolver ${RESOLVER};
+
     location / {
         # Only GET/HEAD
         limit_except GET { deny all; }
         proxy_pass https://ghcr.io;
         proxy_http_version 1.1;
-	# hide upstream header
+        # ghcr.io redirects to an S3 bucket, which is not accessible. So handle
```

Comment:
we trust ghcr.io , so therefore we trust the url we get back?

Reply:
We could hardcode it as per above comment. But it would break if ghcr.io ever change domains on us.